src/System.Text.Json/src/System/Text/Json/Reader/JsonReaderOptions.cs

```diff
@@ -58,8 +62,10 @@ public int MaxDepth
         /// <summary>
         /// Defines whether an extra comma at the end of a list of JSON values in an object or array
         /// is allowed (and ignored) within the JSON payload being read.
-        /// By default, it's set to false, and <exception cref="JsonException"/> is thrown if a trailing comma is encountered.
         /// </summary>
+        /// <remarks>
+        /// By default, it's set to false, and <exception cref="JsonException"/> is thrown if a trailing comma is encountered.
```

Comment:
Nit: "By default, it's set to false" => "The defaults to false"... same for other occurrences